src/extensions/scratch3_vizblocks/index.js

```diff
@@ -1,14 +1,22 @@
+// eslint-disable-next-line max-len
+const assetID = ['64b59074f24d0e2405a509a45c0dadba', '9f75c26aa6c56168a3e5a4f598de2c94', 'e8d8bf59db37b5012dd643a16a636042', '57f7afe3b9888cca56803b73a62e4227', 'b8209e1980475b30ff11e60d7633446d', 'aacb5b3cec637f192f080138b4ccd8d2', '84d9f26050c709e6b98706c22d2efb3d', '6194b9a251a905d0001a969990961724', '55e95fb9c60fbebb7d20bba99c7e9609', '0f53ee6a988bda07cba561d38bfbc36f'];
```

Comment:
Make a comment for this line, something like:
`// Assets of numbers 0-9`

Reply:
added